lib/manager/composer/artifacts.ts

```diff
@@ -129,21 +165,11 @@ export async function updateArtifacts({
           'update ' + updatedDeps.map((dep) => quote(dep.depName)).join(' ')
         ).trim() + ' --with-dependencies';
     }
-    if (config.composerIgnorePlatformReqs) {
```

Comment:
Can you extract the refactor to a separate PR to merge first please.

Reply:
Done, see #12443.